contracts/axelar-gateway/src/contract.rs

```diff
@@ -9,6 +9,15 @@ use crate::{auth, event};
 
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+/// Parameters for extending the contract instance and its instance storage.
+/// 
+/// If the instance's time to live falls below 7 days, it will be extended by 60 days.
+/// 
+/// If at least one message is approved per week, the instance should never be archived.
+const LEDGERS_PER_DAY: u32 = (24 * 3600) / 5;
+const INSTANCE_TTL_THRESHOLD: u32 = 7 * LEDGERS_PER_DAY;
```

Comment:
@milapsheth Just wanted to double check if 7 days would be okay. If there are no message approvals within a week, then we would have to go through an additional recovery process and GMP calls would be blocked in Stellar.

Reply:
increased threshold to 14 days